docs/index.html

```diff
@@ -0,0 +1,250 @@
+<!doctype html>
```

Comment:
Since `index.html` and `loading.gif` are already in the parent folder, I think it's ok to not duplicate it here since you added instructions to the README on how to populate the docs/ folder

Reply:
This ensures that it's deployed on GitHub pages alongside the repository.

Reply:
docs folder removed!